doc/requirements-and-limitations.md

```diff
@@ -30,8 +30,7 @@ The `SUPER` privilege is required for `STOP SLAVE`, `START SLAVE` operations. Th
 
 - MySQL 5.7 `JSON` columns are not supported. They are likely to be supported shortly.
 
-- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table.
-  - As an example, if your table has a single `UNIQUE KEY` and no `PRIMARY KEY`, and you wish to replace it with a `PRIMARY KEY`, you will need two migrations: one to add the `PRIMARY KEY` (this migration will use the existing `UNIQUE KEY`), another to drop the now redundant `UNIQUE KEY` (this migration will use the `PRIMARY KEY`).
+- The two _before_ & _after_ tables must share some `UNIQUE KEY`. Such key would be used by `gh-ost` to iterate the table. See [Read more](shared-key.md)
 
 - The chosen migration key must not include columns with `NULL` values.
```

Comment:
This is great wording and better than I can express myself. Would you mind pushing this into this branch?

Also, I don't know if you've seen the new file, [shared-key.md](https://github.com/github/gh-ost/pull/358/files#diff-a3f29a9d676d9f2703ccd894b444ae48)? It really elaborates on the topic. I'm thinking we can just drop the entire paragraph from the `requirements` page altogether, given that it links to that page, and add your description in the `shared-key.md` file?